Python/flowgraph.c

```diff
@@ -2173,7 +2155,8 @@ remove_unused_consts(basicblock *entryblock, PyObject *consts)
 
     for (basicblock *b = entryblock; b != NULL; b = b->b_next) {
         for (int i = 0; i < b->b_iused; i++) {
-            if (OPCODE_HAS_CONST(b->b_instr[i].i_opcode)) {
+            int opcode = b->b_instr[i].i_opcode;
+            if (OPCODE_HAS_CONST(opcode) && opcode != LOAD_SMALL_INT) {
```

Comment:
`hasconst` is defined in the `dis` docs as "Sequence of bytecodes that access a constant". I think it will make more sense if we clarify that it means "access a constant from `co_consts`" and then remove `LOAD_SMALL_INT` from `hasconst` and `OPCODE_HAS_CONST`.

Reply:
That makes sense. `LOAD_COMMON_CONST` is not in `hasconst`.

Reply:
I don't want to go overboard on testing internal details of the bytecode compiler.
`LOAD_CONST` with a small `int` is still correct, just a bit less efficient than `LOAD_SMALL_INT`.
We already have plenty of tests in `test_dis` that test this